legacy-app/src/app/modules/events/pages/event-details-page/event-details-page.component.html

```diff
@@ -151,6 +147,48 @@
           to seeing you soon!
         </p>
       </mat-expansion-panel>
+      <mat-expansion-panel
+        *ngIf="event.ratings.length > 0 && (eventOver$ | async)"
+        expanded="true"
+      >
+        <mat-expansion-panel-header>
+          <mat-panel-title>Ratings</mat-panel-title>
+        </mat-expansion-panel-header>
+        <div class="mb-6 flex flex-col md:flex-row flex-wrap gap-4">
+          <ng-container *ngFor="let rating of event.ratings">
```

Comment:
This ngFor could be moved down to the app-ratin-item

Reply:
Fixed